### DISCUSSION TITLE
【链接异常】主题切换插件消失，多处链接存在异常

### DISCUSSION THREAD

**Discussion by lifeafter619** [tangly1024/NotionNext#2005] [Q&A]
**描述bug**
1.**在手机浏览器访问时**（电脑上貌似正常），出现了切换主题的插件无法正确显示主题（显示为空白），且无法切换主题。日夜间模式切换，语言切换正常。
目前测试了 heo 和 next 主题，都出现了该问题
![Screenshot_20240210_173011_edit_46450504043633.jpg](https://github.com/tangly1024/NotionNext/assets/65111206/c5345b6f-7743-4f7f-aff4-e5db360339aa)
![Screenshot_20240210_173118_edit_46480543013153.jpg](https://github.com/tangly1024/NotionNext/assets/65111206/4b6b1be1-7f74-49ae-9f87-692820a3ea37)

2.**分享插件**
测试了 heo 主题，文末分享插件获取到的链接异常
如原文链接为 https://66619.eu.org/article/2024-fireworks
分享组件获取的链接为 null/article/2024-fireworks
图为 telegram 分享时获取到的链接
![Screenshot_20240210_175341.jpg](https://github.com/tangly1024/NotionNext/assets/65111206/cd758d59-9a54-4169-aeef-0df04244bd05)

3.**RSS 订阅链接**
站点 feed 页面下可见链接获取异常
![Screenshot_20240210_174121.jpg](https://github.com/tangly1024/NotionNext/assets/65111206/2bf73740-34f0-4692-87ea-0f963d963a0f)
使用第三方获取到的链接也是异常的，如图
![Screenshot_20240210_170953_edit_46460993418056.jpg](https://github.com/tangly1024/NotionNext/assets/65111206/688a55ec-8a7e-45b5-ac1f-26e1aabab5df)

**复现步骤**
手机浏览器浏览站点，最新版本 NotionNext

**期望的正常结果**
正常链接

**截图**
【可选】相关的页面，应该的结果

**环境**

- 【必填】NotionNext版本 [例如. 4.0.18] 最新版本
- 【必填】heo [例如. hexo]
- 【必填】vercel [例如. vercel]
- 【可选】Android [例如. iOS, Android, macOS, windows]
- 【可选】浏览器 Via [例如. chrome, safari, firefox]

**补充说明**
今天大年初一，祝各位新年快乐🎉(＾▽＾)🎉🎇

**Comment by tangly1024** [comment]
hello ， 请尝试更新一下代码

**Comment by lifeafter619** [comment]
主题切换插件已正常，感谢！
但两个链接问题仍然存在

**Comment by lifeafter619** [comment]
![image](https://github.com/tangly1024/NotionNext/assets/65111206/57aa3045-e0af-4c23-b9de-936b8294f224)
